hyperspectral/hyperspectral_workflow.sh

```diff
@@ -110,109 +110,108 @@ drc_envlog='/home/extractor/sites/ua-mac/Level_1/envlog_netcdf'
 
 # When running in a terminal window (not in an non-interactive batch queue)...
 if [ -n "${TERM}" ]; then
-    # Set fonts for legibility
-    fnt_nrm=`tput sgr0` # Normal
-    fnt_bld=`tput bold` # Bold
-    fnt_rvr=`tput smso` # Reverse
+  # Set fonts for legibility
+  fnt_nrm=$(tput sgr0) # Normal
+  fnt_bld=$(tput bold) # Bold
+  fnt_rvr=$(tput smso) # Reverse
 fi # !TERM
 
 # --------------------------------------------------------------------------------------------------------------------------------
 # Defaults for command-line options and some derived variables
 # --------------------------------------------------------------------------------------------------------------------------------
 
-cln_flg='Yes' # [flg] Clean-up (remove) intermediate files before exiting
-dbg_lvl=0 # [nbr] Debugging level
-dfl_lvl='' # [nbr] [enm] Deflate level [0..9]
-drc_in='' # [sng] Input file directory
-drc_in_xmp='drc_in' # [sng] Input file directory for examples
-drc_out="${drc_pwd}" # [sng] Output file directory
-drc_out_xmp='drc_out' # [sng] Output file directory for examples
-flg_swir='No' # [flg] SWIR camera
-flg_vnir='No' # [flg] VNIR camera
+cln_flg='Yes'                                                                                                                                             # [flg] Clean-up (remove) intermediate files before exiting
```

Comment:
Were the comments incorrect? I'm wondering why they were removed

Reply:
they were still there, just pushed to the right of the long 'gaa_sng' flag because intellij formatting likes to align comments. i used the refactoring tool to widely add indenting and make it more readable - moved these comments back to the left.